cpp/include/raft/linalg/lstsq.cuh

```diff
@@ -115,6 +115,143 @@ void lstsqQR(const raft::handle_t& handle,
   detail::lstsqQR(handle, A, n_rows, n_cols, b, w, stream);
 }
 
+/**
+ * @defgroup lstsq Least Squares Methods
+ * @{
+ */
+
+/**
+ * @brief Solves the linear ordinary least squares problem `Aw = b`
+ * Via SVD decomposition of `A = U S Vt` using default cuSOLVER routine.
```

Comment:
Given comments above, it sounds like whether cuSOLVER is used is an implementation detail that we don't have to reveal to users.